manifests/init.pp

```diff
@@ -151,6 +157,15 @@
       key     => $certs::apache::apache_key,
       ca_cert => $certs::ca_cert,
       require => Class['certs::apache'],
+      port    => $docker_registry_port,
+    }
+  }
+
+  if $pulp_master {
+    foreman_config_entry { 'pulp_docker_registry_port':
+      value          => $docker_registry_port,
+      ignore_missing => false,
+      require        => [Exec['foreman-rake-db:seed']],
```

Comment:
This is happening because foreman-rake doesn't exist on a devel install. I am going to propose making this step optional on a devel install (no idea how to do that), then I'll try to adjust the setting in the forklift setup.